running.md

```diff
@@ -134,6 +134,21 @@ sudo firewall-cmd --add-service=cockpit
 sudo firewall-cmd --add-service=cockpit --permanent
 ```
 
+### Fedora CoreOS
+{:#coreos}
+
+The standard Fedora CoreOS image does nto contain any Cockpit packages. To install these, run
```

Comment:
typo: "nto";  should presumably be "not"

Reply:
Thanks David, well spotted! Fixed.